README.md

```diff
@@ -1,6 +1,141 @@
 # ORCID-Model
-Includes the data model and core rules between data structures including the structure for “visibility” (privacy settings)
 
+This repository contains the data model used in the ORCID Registry and APIs as well as Java classes used for mashalling/unmarshalling.
 
-# License
-See [LICENSE.md](https://github.com/ORCID/ORCID-Work-in-Progress/blob/master/LICENSE.md)
+orcid-model is available in Maven central https://mvnrepository.com/artifact/org.orcid/orcid-model/3.0.0
```

Comment:
Hey @lizkrznarich, lets point them to version 3.0.1: https://mvnrepository.com/artifact/org.orcid/orcid-model/3.0.1